docs/integrations/gitops/custom_policy.md

```diff
@@ -0,0 +1,145 @@
+---
+sidebar_position: 3
+title: Write Custom Policies
+---
+
+While Permit.io UI is a good fit for almost every policy configuration need, sometimes developers need to extend their policy configuration to something nonexistent.
+Since Permit.io cloud configuration generates Policy as Code for each user configuration, extending it by writing custom policy code yourself is easy.
+
+This document demonstrates an example of a use-case that is not supported via Permit.io UI but easily possible through Policy as Code extension to the Permit.io UI configuration.
+
+## Deny All Rule
+When a policy is configured in Permit.io's UI, whether it's an RBAC, ABAC, or ReBAC policy, the generated policy rules are Allow rules. This means that if any of the rules allow the user to perform the action, the check function will return true.
+
+In some edge cases, a developer would want to create a special `Deny` rule, which means if a user does not pass a rule, no matter what, they will get `false` due to the `check` function.
+For example, we would like to restrict access to a role `tmp-admin` only to boundaries of time, so even if the configured policy returns `true`, the `Deny` rule will block the access.
+
+## Writing Custom Policy
+Permit.io is based on the open-source policy engine, OPA, and Cedar (TBA more soon) to keep the policy configuration flexible.
+Each engine has a Policy language to write policies, Rego, and Cedar (respectively). This article will show an example of writing custom Rego code to extend Permit.io policy configuration.
+As the first step, you should follow the following guide to enable the GitOps feature that will let you write custom code.
+If you enable the GitOps feature already, continue to the next step to write the custom deny rule.
+
+## The `custom.rego` file
+Permit.io policy repository structure consists two folders, `permit` and `custom`.
+As it might sound from the name, the custom folder is where developers can write the custom Rego code, if you’d look there you’ll find a file named custom.rego with a sample policy script.
+Rego is a logical language (same as Prolog/Datalog), and hence, it could look unnatural for developers who code in common languages, but our rule is quite simple and we will guide step by step.
```

Comment:
```suggestion

_Rego is a logical language (same as Prolog/Datalog), and hence, it could look unnatural for developers who code in common languages, but our rule is quite simple and we will guide step by step._
```